applications/nrf_desktop/src/modules/Kconfig.ble

```diff
@@ -281,7 +281,6 @@ comment "BLE QoS management"
 
 config DESKTOP_BLE_QOS_ENABLE
 	bool "Enable BLE channel map management"
-	depends on BT_CENTRAL
```

Comment:
We should also add USB dependency for QoS stats printout (there are peripherals without USB)